src/list.js

```diff
@@ -0,0 +1,82 @@
+let listArr = JSON.parse(localStorage.getItem('task')) || [];
+export default class List {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+
+  addTask = () => {
```

Comment:
- [ ] Kindly make sure that new tasks use the next index in the list, as for now, all the tasks have `1` as the index. one way to fix this is by setting the index to the length of the array in the local storage, this way the new task will have the proper index
- [ ] Also make sure when deleting a task the indexes of the other task should be changed